### ISSUE TITLE
Sensor data stopped showing up on console on 12/16

### ISSUE THREAD

**Issue by PismoSam** [helium/console#1258]
1. Are you using Helium hosted Console or running your own open source Console? Helium console

2. If you are on Helium hosted Console, which url are you currently on?  https://staging-console.helium.wtf/devices

3. If you are experiencing bugs in Helium hosted Console, please specify your Organization name and Device ID (ex. 3483050b-284d-47e2-8eda-e087fc03a753).  ClearTalk  97bb46af-7e20-44ce-b3e8-2bbe1c346195

Please describe your issue:  I have 7 sensors on console, all stopped showing data on 12/16 after 6pm pacific time. I have plenty of data credits available.

**Comment by PismoSam** [comment]
Let me add that these 7 sensors are spread out throughout northern CA at different locations and can be heard by multiple hotspots. All of the sensors are Dragino LHT65-E1 set to default programming and to report every 20 minutes. All have been working up until that date ad time.

**Comment by lytzV** [comment]
Hi, staging console is not a production-ready environment.
You can the Console Providers [listed here](https://docs.helium.com/use-the-network/console/hosting-providers/).

**Comment by PismoSam** [comment]
Yes, I got that impression from that fact that there was no one to reach out to for help with my problem.  Too bad there are no statements to that effect when you sign up for an account.  I am not using this platform for anything other than to try the Helium Network for what it was built for.
This afternoon as I was getting ready to delete my sensors from Datacake, they all magically started reported again. 

Happy ending?
